test/server/models/Collective.test.js

```diff
@@ -369,6 +369,38 @@ describe('server/models/Collective', () => {
       }
     });
 
+    it('auto cancel pending host applications when host is deactivated', async () => {
+      const temporaryHost = await models.Collective.create({
+        name: 'Temporary Host',
+        slug: 'temporaryHost',
+        hostFeePercent: 0,
+        type: 'ORGANIZATION',
+        currency: 'EUR',
+        CreatedByUserId: user1.id,
+        isHostAccount: true,
+      });
```

Comment:
:ice_cream: Not a requested change: You can use the `fake-data` library for cleaner/simpler tests. Ex. here:

```suggestion
      const temporaryHost = await fakeHot({ admin: user1 });
```

Reply:
Thanks. Done. 👍🏽